cinema/serializers.py

```diff
@@ -69,6 +77,7 @@ class MovieSessionListSerializer(MovieSessionSerializer):
         source="cinema_hall.capacity", read_only=True
     )
     tickets_available = serializers.IntegerField(read_only=True)
+    movie_image = serializers.ImageField(source="movie.title", read_only=True)
```

Comment:
The `movie_image` field is incorrectly sourced. It should be `source="movie.image"` instead of `source="movie.title"` to correctly retrieve the image field from the `Movie` model.